bpf/bpf_lxc.c

```diff
@@ -1941,6 +1941,119 @@ int tail_ipv4_policy(struct __ctx_buff *ctx)
 	return ret;
 }
 
+static __always_inline bool
+ipv4_to_endpoint_is_hairpin_flow(struct __ctx_buff *ctx, struct iphdr *ip4, __u32 __maybe_unused src_sec_identity)
```

Comment:
Note: I haven't given much thought into the naming and location (in the source) for `ipv4_to_endpoint_is_hairpin_flow()`. I guess it's good enough that we can keep it like this, but if you have a better idea don't hesitate to rename it.

Reply:
I think it's fine!